common/app/views/fragments/newHeader.scala.html

```diff
@@ -1,24 +1,10 @@
 @(page: model.Page)(implicit request: RequestHeader)
 
 @import conf.Configuration
-@import common.{LinkTo, NewNavigation, Edition, NavigationHelpers}
+@import common.{LinkTo, Edition}
 @import views.support.RenderClasses
-
-@jobsUrl(edition: String) = {
-    @if(edition == "au") {
-        https://jobs.theguardian.com/landingpage/2868291/jobs-australia-html/?INTCMP=jobs_au_web_newheader
-    } else {
-        https://jobs.theguardian.com?INTCMP=jobs_@{edition}_web_newheader
-    }
-}
-
-@ContributionOrSupporterUrl(edition: String) = {
-    @if(edition == "us") {
-        https://contribute.theguardian.com/us?INTCMP=gdnwb_copts_co_dotcom_header
-    } else {
-        @{Configuration.id.membershipUrl}/@{edition}/supporter?INTCMP=mem_@{edition}_web_newheader
-    }
-}
```

Comment:
This is going straight to my list of reasons why implementing data manipulation method as template block is bad 👿